system.json

```
@@ -85,6 +85,13 @@
       "flags": {}
     }
   ],
+  "packFolders": [{
+      "name": "Delta Green | system",
```

I think just "Delta Green" looks cleaner

```suggestion
      "name": "Delta Green",
```